docs/UserGuide.md

```diff
@@ -28,16 +28,18 @@ as well as summary commands.
 
 4. Double-click the file to start the app. The similar GUI shown below (Figure 1) should appear in a few seconds. <br>
    *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*<br>
-   <br> ![Ui](images/UiWithAnnotation.png) <br>
-   Figure 1: Introduction to OneShelf User Interface
+
+    #<br>![Ui](images/UiWithAnnotation.png) <br>
+    Figure 1: Introduction to OneShelf User Interface
 
 5. Type the command `help start` in the command box and press Enter to execute it. <br>
    A new help window shown below (Figure 2) should appear on your desktop.
-   <br> ![Help Window](images/UiHelpStart.png) <br>
-   Figure 2: Help Start Window
+
+    #<br>![Help Window](images/UiHelpStart.png) <br>
```

Comment:
The spacing for this seems the best thus far 
![image](https://user-images.githubusercontent.com/46394439/97177438-27e73680-17d1-11eb-940d-2c1ba2aff18c.png)